BILLthebuilder/automart
# #166397958 Create endpoint to mark posted ad as sold

* It creates an endpoint for a user to mark a posted ad as sold
#### Description of Task to be completed?
* The `PATCH /api/v2/car/:id/status` endpoint is created here
#### How should this be manually tested?
* Clone the repository
* Checkout to the `ft-mark-as-sold-166397958 ` branch
* Clone the repo
* `cd` into `automart` directory
* Run `npm install` to install package dependencies
* Run `npm test` to run tests
* All the tests should pass
* Run `npm start`
* Run the PATCH `/api/v2/car/:id/status` to view all unsold posted ads
* It should return a `status` of `200` with the updated car ad
#### What are the relevant pivotal tracker stories?
* [#166397958](https://www.pivotaltracker.com/story/show/166397958)
#### Screenshots
![Screenshot from 2019-06-20 16-18-08](https://user-images.githubusercontent.com/24655101/59852234-16895100-9377-11e9-8faa-015298d3bfef.png)
